jdk/src/solaris/bin/java_md_solinux.c

```diff
@@ -380,6 +386,13 @@ CreateExecutionEnvironment(int *pargc, char ***pargv,
                                            asked for? Current model is
                                            fine unless another model
                                            is asked for */
+#ifdef AIX
+    const char *mallocOptionsName = "MALLOCOPTIONS";
+    const char *mallocOptionsValue = "multiheap,buckets";
+    if (setenv(mallocOptionsName, mallocOptionsValue, 0) != 0) {
+      printf("mallocOptins failed\n");
+    }
```

Comment:
The types of the variables have not been updated.
The indentation is not fixed: the `printf` line is not properly indented.
There's still a typo in the message, but `printf` is not an appropriate response to a failure of `setenv`.

Reply:
@keithc-ca what are you thinking should be an appropriate response to a failure? It should never occur, unless you are out of environment space or memory?, but if it does it would be nice to have the message that indicate the JVM will perform differently.

Reply:
If the consensus is that a message is the appropriate response, then, IMO:
* it should be written to `stderr`
* it should speak to the consequences of the failure, e.g. performance may be affected